spec/controllers/collaborators/incomings_controller_spec.rb

```diff
@@ -244,32 +244,17 @@
       let(:request_params) { { id: accepted_collaboration.external_id } }
     end
 
-    it "deletes the collaborator and renders index" do
+    it "deletes the collaborator, sends the appropriate email, and redirects to the collaborations page" do
       expect do
         delete :destroy, params: { id: accepted_collaboration.external_id }
-        expect(response).to have_http_status(:ok)
-        expect(inertia.component).to eq("Collaborators/Incoming/Index")
+        expect(response).to be_redirect
+        expect(response).to redirect_to(collaborators_incomings_path)
         expect(flash[:notice]).to eq("Collaborator removed")
       end.to have_enqueued_mail(AffiliateMailer, :collaboration_ended_by_affiliate_user).with(accepted_collaboration.id)
 
       expect(accepted_collaboration.reload.deleted_at).to be_present
     end
 
-    context "when seller is deleting the collaboration" do
-      before { sign_in seller2 }
-
-      it "deletes the collaborator and sends the appropriate email" do
-        expect do
-          delete :destroy, params: { id: accepted_collaboration.external_id }
-          expect(response).to have_http_status(:ok)
-          expect(inertia.component).to eq("Collaborators/Incoming/Index")
-          expect(flash[:notice]).to eq("Collaborator removed")
-        end.to have_enqueued_mail(AffiliateMailer, :collaboration_ended_by_seller).with(accepted_collaboration.id)
-
-        expect(accepted_collaboration.reload.deleted_at).to be_present
-      end
-    end
-
```

Comment:
Sorry, I forgot to clean this up in my self-review 😅

It's a little bit odd though now, since both routes can delete the collaboration record for a seller and affiliate user. My reason for structuring `detroy` in `base_controller` was to avoid code duplication but LMK if this is un-desired?